media-gfx/ephoto/ephoto-1.5.ebuild

```diff
@@ -1,23 +1,24 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
 DESCRIPTION="Enlightenment image viewer written with EFL"
 HOMEPAGE="https://www.enlightenment.org/about-ephoto"
-SRC_URI="http://www.smhouston.us/stuff/${P}.tar.xz"
+SRC_URI="https://download.enlightenment.org/rel/apps/${PN}/${P}.tar.xz"
 
 LICENSE="BSD-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="nls"
 
-RDEPEND=">=dev-libs/efl-1.18.0[eet,X]"
-DEPEND="
-	${RDEPEND}
-	virtual/pkgconfig
-	nls? ( sys-devel/gettext )
-"
+RDEPEND="dev-libs/efl[eet,X]"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig
+	nls? ( sys-devel/gettext )"
+
+# mismatch with real SRC_URI archive
+RESTRICT="mirror"
```

Comment:
Please drop that. Just rename the tarball in `SRC_URI` in order not to cause mirror failures.

Reply:
Something like `-> ${P}-1.tar.xz`?

Reply:
yes, for example.